framework/src/modules/http_api/schema/swagger.yml

```diff
@@ -2118,6 +2118,8 @@ definitions:
       - publicKey
       - balance
       - nonce
+      - votes
+      - delegate
```

Comment:
I think `keys` and `unlocking` is missing here and in properties

Reply:
Added (given that the code deletes the `keys` property if empty I did not added it as required)